### PR TITLE
MDEXP-279: adding loan type function

### PR DESCRIPTION
**Pull request by KVupp**
https://issues.folio.org/browse/MDEXP-279

## PUROSE
To map the permanent loan type , we need a translation function. This PR contains the translation function

## Reference:
Needs to be merged along with https://github.com/folio-org/mod-data-export/pull/152

### CODE REVIEW

File: src/main/java/org/folio/processor/translations/TranslationsFunctionHolder.java [hunk 1]
```diff
@@ -84,6 +85,30 @@ public String apply(String identifierValue, int currentIndex, Translation transl
       return StringUtils.EMPTY;
     }
   },
+  SET_LOCATION() {
```
Comment (andrii-novytskyi) [inline src/main/java/org/folio/processor/translations/TranslationsFunctionHolder.java:88]: TranslationsFunctionHolder already contains "SET_LOCATION"

**Review by andrii-novytskyi — CHANGES_REQUESTED** [review]

File: src/main/java/org/folio/processor/translations/TranslationsFunctionHolder.java (hunk 1 above)
Comment (illia-borysenko) [reply src/main/java/org/folio/processor/translations/TranslationsFunctionHolder.java:88]: Yes, please remove it, I created new location function to get all related reference data, I'm also going to create quick fix for mod-data-export to make build green

Comment (KVupp) [reply src/main/java/org/folio/processor/translations/TranslationsFunctionHolder.java]: It wasn't added new in this PR, it was already existing, I think it didn't show it as merge conflict from master, fixed that now

**Review by andrii-novytskyi — APPROVED** [review]

**Comment by andrii-novytskyi** [comment]
Please Add unit tests for loan type before merge

**Comment by KVupp** [comment]
> Please Add unit tests for loan type before merge

I had an observation on the unit tests, so I saw that the tests were written to send just the first entry? Is there a reason for that?  we need to send all the values in the reference data for proper tests. I added my test to add all the entries . 
If there was no specific reason, I suggest we change all the other reference data too

**Review by illia-borysenko — APPROVED** [review]

**Pull request merged**